src/main/java/com/spotify/docker/client/messages/ContainerConfig.java

```diff
@@ -372,12 +372,20 @@ public Builder volumes(final String... volumes) {
     @JsonProperty("Retries")
     public abstract Integer retries();
 
+    /**
+     * In nanoseconds.
+     */
+    @Nullable
+    @JsonProperty("StartPeriod")
+    public abstract Long startPeriod();
+
     @JsonCreator
     public static Healthcheck create(
             @JsonProperty("Test") final List<String> test,
             @JsonProperty("Interval") final Long interval,
             @JsonProperty("Timeout") final Long timeout,
-            @JsonProperty("Retries") final Integer retries) {
+            @JsonProperty("Retries") final Integer retries,
+            @JsonProperty("StartPeriod") final Long startPeriod) {
```

Comment:
I don't know if this is a great idea, so maybe @davidxia or @mattnworb could weigh in, but changing this method signature seems too disruptive. Should we add another convenience `create` method with the same signature as the old one (`create(List<String, Long, Long, Integer)`) that just calls this one with a final `(Long) null`? It could even be `@Deprecated`. But that would give users some time to migrate their code before it just breaks on a version bump.

I'm sure this situation has come up in the past, but I don't know what we have done.